### ISSUE TITLE
Error: Replicating claims as headers is deprecated and will removed from v4.0.

### ISSUE THREAD

**Issue by markdynamo6** [Firesphere/silverstripe-graphql-jwt#35]
We are suddenly getting this on a fresh install:

`Replicating claims as headers is deprecated and will removed from v4.0. Please manually set the header if you need it replicated.`

Seems to be an issue with lcobucci/jwt. Is this something you've seen?

Just looked through the pull requests and might be related to https://github.com/Firesphere/silverstripe-graphql-jwt/pull/32#issuecomment-741125296

**Comment by Firesphere** [comment]
Yes, you're probably best off checking in on #32 and/or @tractorcow on this issue
